fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/metadata/MetaDataProtoTest.java

```diff
@@ -300,8 +306,8 @@ public void versionstampIndexDeserialization() throws Exception {
                                                 .setField(scalarField("rec_no"))))));
 
         RecordMetaData metaData = RecordMetaData.newBuilder().addDependencies(BASE_DEPENDENCIES)
-                .setRecords(protoBuilder.build())
-                .getRecordMetaData();
+                .setRecords(protoBuilder.build(), true)
+                .build(false);
```

Comment:
I don't think we want the extension options processed here, but I do think we want the meta-data validated now that #354 is in.